README.md

```diff
@@ -42,23 +42,29 @@ pip install tensorflow-transform
 To build from source follow the following steps:
 Create a virtual environment by running the commands
 
-```
+```bash
 python3 -m venv <virtualenv_name>
 source <virtualenv_name>/bin/activate
 pip3 install setuptools wheel
 git clone https://github.com/tensorflow/transform.git
 cd transform
-python3 setup.py bdist_wheel
+pip3 install .
 ```
 
-This will build the TFT wheel in the dist directory. To install the wheel from
-dist directory run the commands
+If you are doing development on the TFT repo, replace
 
+```bash
+pip3 install .
 ```
-cd dist
-pip3 install tensorflow_transform-<version>-py3-none-any.whl
+
+with
+
+```bash
+pip3 install -e .
 ```
```

Comment:
I'd vote to remove the `3` from `pip3` and `python3` specified here, which is used to distinguish from python 2 - I don't think anyone is running py2 anymore by default, but I could be wrong about that. What do you think?

Reply:
I agree with your suggestion